pkg/kube/inject/testdata/inject/auth.non-default-service-account.yaml.injected

```diff
@@ -87,6 +87,16 @@ spec:
             ]
         - name: ISTIO_META_APP_CONTAINERS
           value: hello
+        - name: GOMEMLIMIT
+          valueFrom:
+            resourceFieldRef:
+              divisor: "0"
+              resource: limits.memory
+        - name: GOMAXPROCS
+          valueFrom:
+            resourceFieldRef:
+              divisor: "0"
```

Comment:
Can you educate me on what divisor 0 does here and elsewhere ? (yet not above?) [on the surface dividing by 0 seems... scary]

Reply:
This is equivilent to unset. I am not 100% the full end to end flow here tbh... I think there must be some custom json marshaling here. In k8s code there is

```
	if divisor.Cmp(fs.Divisor) == 0 {
		divisor = resource.MustParse("1")
```